generative_model_training_all_labels/05_plot_labels_sampling_performance_all_labels.ipynb

```diff
@@ -0,0 +1,1635 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            color="label_source",
</pre><p>Probably can remove this line to remove the legend </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/snorkeling-full-text/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>